airbyte-integrations/connectors/source-mssql/src/main/resources/spec.json

```diff
@@ -0,0 +1,36 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/mysql",
+  "connectionSpecification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "MySql Source Spec",
+    "type": "object",
+    "required": ["host", "port", "database", "username"],
+    "additionalProperties": false,
+    "properties": {
+      "host": {
+        "description": "Hostname of the database.",
+        "type": "string"
+      },
+      "port": {
+        "description": "Port of the database.",
+        "type": "integer",
+        "minimum": 0,
+        "maximum": 65536,
+        "default": 3306,
```

Comment:
is the UI ok with defaults now?

Reply:
it should. artem pushed a fix. but i might remove it for now just to be safe.